projects/storefrontlib/cms-components/navigation/navigation/navigation-ui.component.html

```diff
@@ -31,37 +26,47 @@
       [target]="node.target"
       [style]="node.styleAttributes"
       [class]="node.styleClasses"
-      [isNavLink]="true"
+      (click)="closeIfClickedTheSameLink(node)"
     >
       {{ node.title }}
-      <cx-icon
-        *ngIf="flyout && node.children?.length > 0"
-        [type]="iconType.CARET_DOWN"
-      ></cx-icon>
     </cx-generic-link>
 
     <ng-template #heading>
-      <span [attr.tabindex]="flyout && (depth === 0 || node.url) ? 0 : -1">
-        {{ node.title }}
-        <cx-icon
-          *ngIf="flyout && node.children?.length > 0"
-          [type]="iconType.CARET_DOWN"
-        ></cx-icon>
-      </span>
+      <ng-container *ngIf="flyout && node.children?.length > 0; else span">
```

Comment:
`; else span` -> for me, I would recommend using different word rather than a tag name